### PR TITLE
feat: issue template for CI environment

### PR DESCRIPTION
**Pull request by matebarabas**
This PR adds a new issue template for CI environment related questions/suggestions/feedback.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: AVM - CI Environment Issue 🚀
+description: Have an issue, question or suggestion for the CI Environment? Let us know!
+title: "[AVM CI Environment Issue]: "
+labels: ["Type: AVM :a: :v: :m:", "Needs: Triage :mag:"]
```
Comment (eriqua) [inline .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:4]:
```suggestion
labels: ["Type: AVM :a: :v: :m:", "Needs: Triage :mag:", "Needs: Core Team :genie:"]
```

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        > **NOTE**: If your question/request is related to the AVM site/documentation, please file an issue in the [AVM repo](https://github.com/Azure/Azure-Verified-Modules/issues/new?assignees=&labels=Type%3A+Question%2FFeedback+%3Araising_hand%3A%2CNeeds%3A+Triage+%3Amag%3A&projects=&template=question_feedback.yml&title=%5BQuestion%2FFeedback%5D%3A+).
+  - type: checkboxes
+    id: existing-checks
+    attributes:
+      label: Check for previous/existing GitHub issues
+      description: By submitting this issue, you confirm that you have [searched for previous/existing GitHub issues](https://aka.ms/BRM/TypeAVM) to avoid creating a duplicate.
+      options:
+        - label: I have checked for previous/existing GitHub issues
+          required: true
+  - type: textarea
```
Comment (eriqua) [inline .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:21]: Similarly to module issues, they can be either bugs or feature requests
```suggestion
  - type: dropdown
    id: issue-type
    attributes:
      label: Issue Type?
      description: How would you best describe this issue? Is this a...
      options:
        - ""
        - "Feature Request"
        - "Bug"
    validations:
      required: true
      - type: textarea
```

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml (hunk 1 above)
Comment (eriqua) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:4]: CI label to be added too as soon as created

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml (hunk 2 above)
Comment (matebarabas) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:21]: This is a good suggestion. We'll have to add the related automation too though.

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml (hunk 1 above)
Comment (matebarabas) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:4]: Added that too. 👍

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml (hunk 2 above)
Comment (matebarabas) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:21]: ...which I just checked and it should be working the way it's defined, so we're good.

**Comment by matebarabas** [comment]
@eriqua, this is ready for final review. @AlexanderSehr, please also take a look and share your feedback! Thanks!

File: .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+name: AVM - CI Environment Issue 🚀
+description: Have an issue, question or suggestion for the CI Environment? Let us know!
```
Comment (AlexanderSehr) [inline .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:2]: @eriqua, do you think it also makes sense to call out `utilities` seperately, or should we consider the 'CI Environment' as an overarching term for all things non-module-related?

Comment (eriqua) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:2]: Nothing against being more specific and mentiong "CI Environment or utilities" in the issue description, similarly to what already done in PR templates

Comment (matebarabas) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:2]: @AlexanderSehr, @eriqua, 
- would you only change the description to: "Have an issue, question or suggestion for the CI Environment or its utilities? Let us know!"
- OR
- would you also change the template's name to "AVM - CI Environment or Utility Issue 🚀" AND the title to "[AVM CI Environment or Utility Issue]: "

My 2 cents: now that we have the concept of "utility modules" mentioning utilities in this context might be misleading (though we all understand that it's not the same thing).

Comment (AlexanderSehr) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:2]: I prefer the former as it does avoid confusion.

If we were to go with the latter, then I'd suggest to call it something like `CI-Utility`.

Comment (eriqua) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml:2]: Good point Mate let's not mix them up. +1 for the former

Comment (eriqua) [reply .github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml]: Lgtm. I'll leave @AlexanderSehr the honors as he opened this thread.

**Review by AlexanderSehr — APPROVED** [review]

**Comment by AlexanderSehr** [comment]
@matebarabas, merge at will

**Pull request merged**